Doc/library/http.client.rst

```diff
@@ -42,6 +43,8 @@ The module provides the following classes:
    (if it is not given, the global default timeout setting is used).
    The optional *source_address* parameter may be a tuple of a (host, port)
    to use as the source address the HTTP connection is made from.
+   The optional *blocksize* parameter sets the buffer size used to send a
```

Comment:
Please add "in bytes" to size.

Reply:
Thanks for the reivew, will update in the next version.

I'll check how to add a test for this.